docs/blog/write-the-docs-2019-stats.rst

```diff
@@ -0,0 +1,99 @@
+.. post:: Jan 10, 2020
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs 2019 Stats
+=========================
+
+Since it's quiet over the holidays,
+we normally don't do a newsletter in January.
+Instead,
+we send out a yearly update for the Write the Docs community,
+filling you in on our progress.
+
+Community Updates
+-----------------
+
+We have a few outstanding things that we wanted to update you on,
+coming into 2020.
+
+The first big change is the `Write the Docs Enhancement Proposals (WEPs) <https://github.com/writethedocs/weps/>`_ process that we have implemented.
+We have approved WEP0_,
+which defines the process,
+and we will be running more proposals through this process soon.
+We hope that you will provide your feedback and insight to us as we look to guide the community into the future.
+
+The second major thing is :doc:`another US conference </conf/cfp/east-2020>` closer to the east coast.
+We've still in the process of talking with venues and trying to see if will work for 2020.
+We will let everyone know the status of the event _before_ Portland tickets sell out,
+so that you won't miss your chance to attend.
+
+The last major thing is our :doc:`Salary Survey </surveys/salary-survey-sep-2019>`,
+which finished up in November.
+This is the first year that we've done this,
+and we were blown away by the response.
+We had over 700 people fill it out,
+and we're in the process of sifting through the data to provide some insights.
+We plan to release the full results soon,
+and in the future hopefully we will have a smoother process for this in place to get it turned around faster.
+
+.. note:: 
+   
+   You can also read our posts from 2016_, 2017_, and 2018_.
+
+.. _WEP0: https://github.com/writethedocs/weps/blob/master/accepted/WEP0000.rst
+.. _2018: https://www.writethedocs.org/blog/write-the-docs-2018-stats/
+.. _2017: https://www.writethedocs.org/blog/write-the-docs-2017-stats/
+.. _2016: https://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/
+
+Conferences
```

Comment:
I'd add a "2019 Statistics" heading here, and bump all the following headings one level down so that they're nested under that heading.